astrokat/simulate.py

```diff
@@ -312,6 +314,8 @@ def _target_azel(self, target):
 
         """
         az, el = target.azel(simobserver.date)
+#         az, el = target.azel(timestamp=simobserver.date,
+#                              antenna=ref_antenna)
```

Comment:
did we mean to comment these 2 lines?

Reply:
Yes, I removed the dead code in the script -- the update will deviate from the system's expected input and thus break the code